test/lib/completions/alias.exp

```diff
@@ -30,8 +30,8 @@ set cmd "alias foo"
 append cmd \002\002\002; # \002 = ^B = Move cursor left in bash emacs mode
 send "$cmd\t"
 expect {
-    -re "^alias foo\r\nbar +foo *\r\n/@alias foo$" { pass "$test" }
-    -re "^alias foo=$" { fail "$test: Wrong cursor position" }
+    -re "^alias foo(\b\b\b)?\r\nbar +foo *\r\n/@alias foo(\b\b\b)?$" { pass "$test" }
+    -re "^alias (foo\b\b\b)?foo=(foo\b\b\b)?$" { fail "$test: Wrong cursor position" }
```

Comment:
While this seems to fix all cases I'm aware of, I'd rather like to understand when exactly does the \b get printed, and when it works as the backspace (i.e. not printed). Do you have any further details? The data I have is:

- My Fedora 25 local box, bash 4.3.43: _not_ printed
- bash-completion fedoradev container, bash 4.4.11: _not_ printed
- bash-completion centos6 container, bash 4.1.2: printed
- bash-completion ubuntu14 container, bash 4.3.11: printed